src/include/sof/audio/audio_stream.h

```diff
@@ -46,15 +46,23 @@ struct sof_audio_stream_params {
 	uint32_t rate;		/**< Number of data frames per second [Hz] */
 	uint16_t channels;	/**< Number of samples in each frame */
 
-	/** alignment limit of stream copy, this value indicates how many
-	 * integer frames which can meet both byte align and frame align
-	 * requirement. it should be set in component prepare or param functions
+	/**
+	 * Alignment limit of stream copy, used by audio_stream_avail_frames_aligned.
+	 * We can set byte_align and frame_align in the init or prepare functions to
+	 * limit the processing data size is byte_align bytes aligned and frame_align
+	 * frames aligned. This feature indicates how many integer frames can meet both
```

Comment:
feature or number or data or value is better?

Reply:
that comment is not helpful @andrula-song. It's not clear if 'byte_align' is a boolean or a number of bytes to align on. Likewise 'frame_aligned' could be a boolean but you seem to use it as a counter. 

Consider renaming the structure members to describe exactly what they do in self-explanatory ways.